src/core/server/saved_objects/mappings/types.ts

```diff
@@ -30,6 +30,8 @@
  * GitHub history for details.
  */
 
+import type { opensearchtypes } from '@opensearch-project/opensearch';
```

Comment:
Is there a particular reason why the name is all lowercase?

Reply:
Hey @tmarkley , we import opensearchtypes from [here](https://github.com/opensearch-project/opensearch-js/blob/68eea00b078710c876bd4c65f7b817c8d418a25f/api/new.d.ts#L698) .

When we fork elasticsearch-js, we followed the same naming rules. So es is changed to opensearch.

```
export * as estypes from './types'
```

```
export * as opensearchtypes from './types'
```
I see your point, seems `opensearchTypes` is better. For now, you need to use `opensearchtypes`.

Reply:
I think after v2.0 we won't maintain nodejs client. We could open an issue. What do you think?

Reply:
I agree we should create an issue still. And perhaps we are talking v2 for the OpenSearch JS client?

Reply:
Create an issue first in opensearch-js repo: https://github.com/opensearch-project/opensearch-js/issues/206